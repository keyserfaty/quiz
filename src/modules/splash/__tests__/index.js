import React from 'react'
import Splash from '../index'

import Shallow from 'react-test-renderer/shallow'

const renderer = new Shallow()

test('Should render', () => {
  const component = renderer.render(<Splash />)
  expect(component).toMatchSnapshot()
})