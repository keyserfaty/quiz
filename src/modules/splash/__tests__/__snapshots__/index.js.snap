// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render 1`] = `
<Component
  style={
    Object {
      "backgroundColor": "#37AFFF",
      "flex": 1,
    }
  }
>
  <Component
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
        "marginTop": 20,
      }
    }
  >
    <Component>
      Questions!
    </Component>
  </Component>
</Component>
`;
